static/js/modules/tables.js

```diff
@@ -485,12 +487,44 @@ DataTable.prototype.ensureWidgets = function() {
     this.$exportWidget = $(exportWidgetTemplate(title));
     $paging.after(this.$exportWidget);
     this.$exportButton = $('.js-export');
-    this.$exportWidget.on('click', this.export.bind(this));
+    this.$exportTooltipContainer = $('.js-tooltip-container');
+    this.$exportButton.on('click', this.export.bind(this));
   }
 
   this.hasWidgets = true;
 };
 
+DataTable.prototype.disableExport = function() {
```

Comment:
These methods need tests. I'm fine doing a little manual QA and merging so we can test this with users if that would be useful, but we'll want tests before this gets into develop.